runners/direct-java/pom.xml

```diff
@@ -74,6 +74,9 @@
                 <dependency>org.apache.beam:beam-sdks-java-core</dependency>
                 <dependency>org.apache.beam:beam-runners-core-java</dependency>
               </dependenciesToScan>
+              <excludes>
+                <exclude>org.apache.beam.sdk.transforms.OldDoFnTest</exclude>
```

Comment:
There are some overspecified tests of aggregators that we don't need to support. I'll see if I can isolate what is wrong with them.